.github/workflows/ci.yaml

```diff
@@ -0,0 +1,19 @@
+on: push
```

Comment:
I think you need to add pull_request to run CI on this PR.

Reply:
Done, though I don't think there's any way to make the CI run for this PR in particular.